src/components/UserMenu/ManaBalances/ManaBalances.styled.tsx

```diff
@@ -1,18 +1,85 @@
 import styled from "@emotion/styled"
-import { Typography } from "@mui/material"
+import { Button, Typography } from "@mui/material"
 
 const ManaBalancesWrapper = styled("div")((props) => {
   const { theme } = props
   return {
     display: "flex",
+    flexDirection: "column",
+    gap: "4px",
+    alignItems: "flex-end",
+    fontSize: theme!.typography.subtitle2.fontSize,
     "& svg": {
       fontSize: theme!.typography.subtitle2.fontSize,
     },
+    "& > div": {
+      display: "flex",
+      flexDirection: "row",
+      alignItems: "center",
+    },
+  }
+})
+
+const GetCreditsButton = styled(Button)((props) => {
+  const { theme } = props
+  return {
+    "&.MuiButton-root.MuiButton-textPrimary": {
+      fontSize: theme!.typography.caption.fontSize,
+      padding: 0,
+      marginRight: "4px",
+      "&:hover": {
+        backgroundColor: "transparent",
+      },
+    },
+    "&:hover": {
+      backgroundColor: "transparent",
+    },
   }
 })
 
 const ManaBalanceNumber = styled(Typography)({
   lineHeight: "inherit",
 })
 
-export { ManaBalancesWrapper, ManaBalanceNumber }
+const CreditsBalanceContainer = styled("div")({
```

Comment:
What do you think about changing div for Box
```suggestion
const CreditsBalanceContainer = styled(Box)({
```